updates/accounting_rules/v4.json

```diff
@@ -62,6 +62,24 @@
             "count_entire_amount_spend": false,
             "count_cost_basis_pnl": true,
             "accounting_treatment": null
+        },
+        {
+            "event_type": "adjustment",
+            "event_subtype": "spend",
+            "counterparty": null,
+            "taxable": false,
```

Comment:
This two should be taxable. We have in kraken's comments that is where we use it mostly

```
                        # Lefteris has seen it in: Crediting airdrops/fork coins
                        # such as ETC, BCH, BSV. OR the XXLM airdrop. Also for forced
                        # removal of a coin due to delisting(negative amount), which is followed
                        # by another similar transfer with positive amount. Also OTC
                        # or other private deals seem to have this type
```